governance/SDK.md

```diff
@@ -0,0 +1,172 @@
+## SDK Community
+
+The community is organized as follows:
+
+- Every SDK has its own
+  [Github Team](https://github.com/orgs/cloudevents/teams), for example
+  `sdk-go-maintainers` is the group of mantainers for
+  [cloudevents/sdk-go project](https://github.com/cloudevents/sdk-go)
+- The union of all the `sdk-*-maintainers` assembles the _sdk maintainers_ group
+
+## New maintainers
+
+TBD
+
+## Ensuring projects health
+
+In order to ensure the health of the SDK projects, we define the following
+actions. These are crucial to show the commitment of the community to develop
+and support these libraries, which are an important part of the CloudEvents
+ecosystem.
+
+We define an SDK project _healthy_ if:
+
+1. It works with the latest version of the programming language
+2. It supports the latest versions of the integrated libraries/frameworks
+3. It receives security patches regularly
```

Comment:
```suggestion
3. Any security patches received are applied and the library is published regularly
```
This phrasing is a little odd. What if there are no security issues with the library? Slightly modified to address that.

Reply:
If there are no security issues and all the other criteria are respected, then there is no need to publish regularly.
Your phrasing seems like saying "even if you meet 1, 2 and there is no need of security patches, library should be published regularly" which doesn't make sense to me honestly. If the library works, it's not buggy, it's secure and new 3rd party integrations can interact with it, there is no need to publish it.

Reply:
OK, how about this?

```
3. Any security patches received are applied and published within one month of receipt
```

Reply:
but then it seems similar to criteria 8... Maybe should I remove one of the two?